docs/src/manual/containers.md

```diff
@@ -485,3 +485,142 @@ JuMP.Containers.SparseAxisArray{Int64, 2, Tuple{Int64, Int64}} with 2 entries:
   [1, 2]  =  3
   [1, 4]  =  5
 ```
+
+## Keyword indexing
+
+JuMP v1.10.0 added experimental support for keyword indexing of
+[`Containers.DenseAxisArray`](@ref) and [`Containers.SparseAxisArray`](@ref).
+Keyword indexing lets you uses the named indices of [`Containers.DenseAxisArray`](@ref)
+and [`Containers.SparseAxisArray`](@ref) as keyword arguments when indexing a
+container.
+
+### The container macro
+
+For containers constructed using the [`Containers.@container`](@ref) macro,
+opt-in to the keyword indexing syntax by passing
+`enable_keyword_indexing = true`:
+
+```jldoctest
+julia> Containers.@container(
+           x[i=2:4, j=1:3],
+           i + j,
+           enable_keyword_indexing = true,
+       )
+2-dimensional DenseAxisArray{Int64,2,...} with index sets:
+    Dimension 1, 2:4
+    Dimension 2, Base.OneTo(3)
+And data, a 3×3 Matrix{Int64}:
+ 3  4  5
+ 4  5  6
+ 5  6  7
+
+julia> x[i = 2, j = 1]
+3
+
+julia> Containers.@container(
+           y[i=2:4, j=1:3; i > j],
+           i + j,
+           enable_keyword_indexing = true,
+       )
+JuMP.Containers.SparseAxisArray{Int64, 2, Tuple{Int64, Int64}} with 6 entries:
+  [2, 1]  =  3
+  [3, 1]  =  4
+  [3, 2]  =  5
+  [4, 1]  =  5
+  [4, 2]  =  6
+  [4, 3]  =  7
+
+julia> y[i = 2:4, j = 1]
+JuMP.Containers.SparseAxisArray{Int64, 1, Tuple{Int64}} with 3 entries:
+  [2]  =  3
+  [3]  =  4
+  [4]  =  5
+```
+
+### The JuMP macros
+
+For containers constructed using the JuMP macros like [`@variable`](@ref),
+opt-in using [`enable_container_keyword_indexing`](@ref):
```

Comment:
Why not `enable_keyword_indexing` ? It's clear we refer to containers when we say `indexing` isn't it ?